clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3545,6 +3545,30 @@ scope) variables and static local variable as well.
   }];
 }
 
+def OpenCLAddressSpaceGlobalDeviceDocs : Documentation {
+  let Category = DocOpenCLAddressSpaces;
+  let Heading = "[[clang::opencl_global_device]]";
+  let Content = [{
+The global_device address space attribute specifies that an object is allocated
+in global memory on device. It is supposed to be used only in SYCL headers and
+not in the actual OpenCL/SYCL user code. It helps distinguishing USM pointers
```

Comment:
It is strange that attributes named `OpenCLAddressSpace...` can appear only in **SYCL** headers. I like the idea with `usm*` naming.